javascript-local-storage/data.js

```diff
@@ -0,0 +1,15 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javscript-local-storage');
+
+todos = JSON.parse(previousTodosJSON);
+
```

Comment:
You'll want to use a conditional to check if `getItem` returned `null` or not. You can test that you code works properly by opening an In Cognito / Private Browsing window.

Reply:
how and why would it be better to open the code in an incognito browser? I've never done that before

the only options i have in VS Code are 'open in default' which is what i use all the time
or open in other browser